README.md

```diff
@@ -1,3 +1,112 @@
 # adage-server
 
 [![Code Climate](https://codeclimate.com/github/greenelab/adage-server/badges/gpa.svg)](https://codeclimate.com/github/greenelab/adage-server)
+
+## Get a working instance of the adage-server running:
+
+### Clone the adage-server repository
+```shell
+git clone git@github.com:greenelab/adage-server.git
```

Comment:
If developers will be submitting pull requests, isn't it easier to make a personal fork first and then clone that? The README should walk people through it the way we want them to do it.